starknet_py/class_hash.py

```diff
@@ -0,0 +1,124 @@
+import json
+from typing import List
+
+from starkware.cairo.common.hash_state import compute_hash_on_elements
+from starkware.starknet.public.abi import starknet_keccak
+
+from starknet_py.cairo.felt import encode_shortstring
+from starknet_py.constants import API_VERSION
+from starknet_py.net.client_models import DeclaredContract, EntryPoint
+
+
+def compute_class_hash(contract_class: DeclaredContract) -> int:
+    """
+    Calculate class hash of a declared contract.
```

Comment:
What exactly do you mean by a not declared contract?
We do not have a `ContractClass` currently in the code. We do have `DeclaredContract` that is just a dataclass with the same 3 fields (program, entry_points_by_type, abi) as ContractClass from cairo-lang.

Reply:
ContractClass from cairo-lang would be used only in this place. Is there a point in creating a **ContractClass** of our own (since we don't want to depend on cairo-lang anymore) and have it's fields defined now (when changes to it are coming)? Maybe we could just go with **TypedDict ContractClass** for now?

Reply:
Unfortunately, it will be used in the `transactions.py`. `Declare` needs it